stdlib/unittest/_log.pyi

```diff
@@ -2,15 +2,15 @@ import logging
 import sys
 from types import TracebackType
 from typing import ClassVar, Generic, NamedTuple, TypeVar
-from unittest.case import TestCase
+from unittest.case import TestCase, _AssertRaisesBaseContext
 
 _L = TypeVar("_L", None, _LoggingWatcher)
 
 class _LoggingWatcher(NamedTuple):
     records: list[logging.LogRecord]
     output: list[str]
 
-class _AssertLogsContext(Generic[_L]):
+class _AssertLogsContext(_AssertRaisesBaseContext, Generic[_L]):
```

Comment:
In Python 3.11 and 3.12 at least, this derives directly from `_BaseTestCaseContext`, not `_AssertRaisesBaseContext`.

Reply:
It's that way on all versions - best I can figure is a copy/paste error of some kind. Fixed now, at any rate.